app/services/dttp/contact_update.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Dttp
+  class ContactUpdate
+    include ServicePattern
+
+    class Error < StandardError; end
+
+    attr_reader :trainee, :trainee_creator_dttp_id, :batch_request
+
+    def initialize(trainee:, trainee_creator_dttp_id:)
+      @trainee = trainee
+      @trainee_presenter = Dttp::TraineePresenter.new(trainee: trainee)
+      @trainee_creator_dttp_id = trainee_creator_dttp_id
+    end
+
+    def call
```

Comment:
Should these be done as a batch operation?

Reply:
We'll be putting this into a job queue, so hopefully the sequencing will be kept intact. If we're running this as an overnight job, it should be alright - but if we're updating the instant a field gets changed, then we might have some issues. Hard to say without the use case

Reply:
I think the queue is the issue. If the second operation fails the first will be redone. It's probably not a big deal apart from the potential for subsequent jobs to get out of order. We can think about that when we do the queueing.